crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -280,6 +280,11 @@ impl SourceMapLookup {
     }
 }
 
+// TODO: Handle relative paths like `../`, `./`, etc
+fn join_url(base: &Url, url: &str) -> Option<Url> {
```

Comment:
Should we pull this util from the sentry-cli @loewenheim?

Reply:
We can also use https://docs.rs/normpath/latest/normpath/ instead to make it more battletested

Reply:
I'm not sure that crate does exactly what we need here. At least in `sentry-cli`, I don't think we want to normalize leading `.`s away. Taking the function from `sentry-cli` could work, but it doesn't operate on URLs.